### PR TITLE
feat: [console-lite] - refactor wallet connector

### LINKED ISSUE #1125: [console-lite] Wallet connector refactoring
## Task
Make wallet connector more universal. Unificate it across all uses

## ACs
- [ ] Make separate component from trading page
- [ ] Use the component in portfolio page

### PR DESCRIPTION
**Pull request by macqbat**
# Related issues 🔗

Closes #1125 

# Description ℹ️

Make wallet connector more universal. Unificate it across all uses

# Demo 📺

image

image

# Technical 👨‍🔧

Make separate component from trading page. Use the component in portfolio page

### CODE REVIEW

File: apps/console-lite/src/app/components/wallet-connector/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      </Button>
+      <h4 className="text-lg text-offBlack dark:text-white">
+        {t("Don't have a wallet?")}
+      </h4>
+      <p>
+        {t('Head over to ')}
+        <a className="text-blue" href="https://vega.xyz/wallet">
+          https://vega.xyz/wallet
+        </a>
+        {t(' and follow the steps to create one.')}
```
Comment (dexturr) [inline apps/console-lite/src/app/components/wallet-connector/index.tsx:34]: thought (non-blocking): outside the scope of this PR but I think we need somthig better for string interporation.

**Review by dexturr — APPROVED** [review]

**Review by MadalinaRaicu — APPROVED** [review]

**Review by notbot00 — APPROVED** [review]

File: libs/deal-ticket/src/components/market-selector.tsx [hunk 2]
```diff
@@ -116,6 +116,7 @@ export const MarketSelector = ({ market, setMarket, ItemRenderer }: Props) => {
       market: MarketNames_markets,
       index: number
     ) => {
+      event.preventDefault();
```
Comment (asiaznik) [inline libs/deal-ticket/src/components/market-selector.tsx:119]: suggestion (non-blocking / a11y): `preventDefault` here will block normal `[tab]`bing behaviour - consider moving it to the 'allowed keys' if needed

Comment (macqbat) [reply libs/deal-ticket/src/components/market-selector.tsx]: Handling Tab press on suggest list has been added

File: apps/console-lite/src/app/components/wallet-connector/index.tsx (hunk 1 above)
Comment (macqbat) [reply apps/console-lite/src/app/components/wallet-connector/index.tsx:34]: :+1:

**Pull request merged**